data/welcome.txt

```diff
@@ -0,0 +1,5 @@
+ __    __     _                                  _
```

Comment:
Great work with the menu presentation. Considered the importance of having a fancy menu for customer's takeawat